classes/CMSCategory.php

```diff
@@ -612,49 +685,66 @@ public function updatePosition($way, $position)
 			SET `position`= `position` '.($way ? '- 1' : '+ 1').'
 			WHERE `position`
 			'.($way
-                ? '> '.(int)$moved_category['position'].' AND `position` <= '.(int)$position
-                : '< '.(int)$moved_category['position'].' AND `position` >= '.(int)$position).'
-			AND `id_parent`='.(int)$moved_category['id_parent'])
+                ? '> '.(int) $movedCategory['position'].' AND `position` <= '.(int) $position
+                : '< '.(int) $movedCategory['position'].' AND `position` >= '.(int) $position).'
+			AND `id_parent`='.(int) $movedCategory['id_parent'])
         && Db::getInstance()->execute('
 			UPDATE `'._DB_PREFIX_.'cms_category`
-			SET `position` = '.(int)$position.'
-			WHERE `id_parent` = '.(int)$moved_category['id_parent'].'
-			AND `id_cms_category`='.(int)$moved_category['id_cms_category']));
+			SET `position` = '.(int) $position.'
+			WHERE `id_parent` = '.(int) $movedCategory['id_parent'].'
+			AND `id_cms_category`='.(int) $movedCategory['id_cms_category']));
     }
 
-    public static function cleanPositions($id_category_parent)
+    /**
+     * @param int $idCategoryParent
+     *
+     * @return bool
+     */
+    public static function cleanPositions($idCategoryParent)
     {
         $result = Db::getInstance()->executeS('
 		SELECT `id_cms_category`
 		FROM `'._DB_PREFIX_.'cms_category`
-		WHERE `id_parent` = '.(int)$id_category_parent.'
+		WHERE `id_parent` = '.(int) $idCategoryParent.'
 		ORDER BY `position`');
         $sizeof = count($result);
         for ($i = 0; $i < $sizeof; ++$i) {
             $sql = '
 			UPDATE `'._DB_PREFIX_.'cms_category`
-			SET `position` = '.(int)$i.'
-			WHERE `id_parent` = '.(int)$id_category_parent.'
-			AND `id_cms_category` = '.(int)$result[$i]['id_cms_category'];
+			SET `position` = '.(int) $i.'
+			WHERE `id_parent` = '.(int) $idCategoryParent.'
+			AND `id_cms_category` = '.(int) $result[$i]['id_cms_category'];
             Db::getInstance()->execute($sql);
         }
+
         return true;
```

Comment:
Yeah, me too. But I don't want to make huge changes at once anymore. The first Cart PR was a total failure haha (expected `12`, got `0.0000`, errrr...).